packages/node_modules/@webex/common/src/metrics.js

```diff
@@ -0,0 +1,11 @@
+
+// Metrics for service catalog
+export const SERVICE_METRICS = {
+  JS_SDK_SERVICE_NOT_FOUND: 'JS_SDK_SERVICE_NOT_FOUND'
+};
+
+// Metric to do with WDM registration
+export const WDM_METRICS = {
+  JS_SDK_WDM_REGISTRATION_SUCCESSFULL: 'JS_SDK_WDM_REGISTRATION_SUCCESSFULL',
+  JS_SDK_WDM_REGISTRATION_FAILED: 'JS_SDK_WDM_REGISTRATION_FAILED'
+};
```

Comment:
This is amazing to see, This also fulfills a request i believe may exist in our backlog around getting usage/failure metrics from SDK explicit items.